src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.exceptions;
+import wolox.training.models.Book;
+
+public class BookAlreadyOwnedException extends RuntimeException {
+    public BookAlreadyOwnedException(Book book) {
+        super("The book " + book.getTitle() + " is already in the user's collection");
```

Comment:
Why not pass the title instead of the entire book? And you can also store the error as a constant, as  I mentioned in a a different PR 😄
`public static final String BOOK_ALREADY_IN_COLLECTION =  "The book %s is already in the user's collection"`
`String.format(ErrorConstants.BOOK_ALREADY_IN_COLLECTION, title)`